capybara-202203251046516252931676.html

```diff
@@ -0,0 +1 @@
+Make your choice Charlotte
```

Comment:
I wonder if there's a way to get these to go into a .gitignore folder?

Reply:
There is! something like this should do it:
```
capybara-*.html
```